bigtable/admin/table_admin_test.cc

```diff
@@ -0,0 +1,137 @@
+//   Copyright 2017 Google Inc.
```

Comment:
Too many spaces after `//` in this file as well.

Reply:
Fixed.  How did that happen?